lib/token-translator.js

```diff
@@ -123,8 +123,9 @@ TokenTranslator.prototype = {
                  type === tt.arrow || type === tt.jsxTagStart ||
                  type === tt.incDec || type === tt.starstar ||
                  type === tt.jsxTagEnd || (type.binop && !type.keyword) ||
-                 type.isAssign) {
-
+                 type.isAssign ||
+                 (type.prefix && (token.value === "~" || token.value === "!"))
```

Comment:
I looked through the acorn code and I'm pretty sure, that currently if `type.prefix` is true, the value is either `!` or `~`. I only added the value comparison, so that in future if acorn adds another prefix (which might not be a Punctator?) it is not immediately converted to Punctator.

Do you think it is safe to assume that prefix is always a Punctator?